src/js/components/dashboard/table/DashboardTableModal.jsx

```diff
@@ -0,0 +1,155 @@
+/**
+ * DashboardTableModal.jsx
+ * Created by Alisa Burdeyny 11/13/19
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash';
+import Modal from 'react-aria-modal';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import RadioGroup from 'components/SharedComponents/RadioGroup';
+
+const propTypes = {
+    data: PropTypes.object,
+    downloadFile: PropTypes.func.isRequired,
+    closeModal: PropTypes.func.isRequired,
+    isOpen: PropTypes.bool
+};
+
+const defaultProps = {
+    data: {},
+    isOpen: false
+};
+
+export default class DashboardTableModal extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            fileLabel: ''
+        };
+
+        this.updateFileLabel = this.updateFileLabel.bind(this);
+        this.downloadFile = this.downloadFile.bind(this);
+    }
+
+    componentDidMount() {
+        if (this.props.data.fileTypes.length === 1) {
+            this.updateFileLabel(this.props.data.fileTypes[0]);
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (!isEqual(this.props.data, prevProps.data)) {
+            if (this.props.data.fileTypes.length === 1) {
+                this.updateFileLabel(this.props.data.fileTypes[0]);
+            }
+            else {
+                this.updateFileLabel('');
+            }
+        }
+    }
+
+    updateFileLabel(fileLabel) {
+        this.setState({
+            fileLabel
+        });
+    }
+
+    downloadFile() {
+        this.props.downloadFile(this.state.fileLabel, this.props.data.submissionId);
+    }
+
+    render() {
+        let fileList = null;
+        if (this.props.data.fileTypes.length > 1) {
+            const radioColumn = this.props.data.fileTypes.map((fileType) => (
+                {
+                    value: fileType,
+                    label: `File ${fileType}`
+                }
+            ));
+            fileList = (
+                <div>
+                    <h5>File Name</h5>
+                    <RadioGroup
+                        onChange={this.updateFileLabel}
+                        currentValue={this.state.fileLabel}
+                        columns={[radioColumn]}
+                        pageSection="modal" />
+                </div>);
+        }
+
+        // adding this because the linter doesn't like when we just pass true
+        const trueProps = true;
+
+        return (
+            <Modal
+                mounted={this.props.isOpen}
+                onExit={this.props.closeModal}
+                verticallyCenter={trueProps}
```

Comment:
To pass these props as true, you can include them without setting them equal to anything (just `verticallyCenter`)